src/universal/modules/userDashboard/containers/CreditCardModal/CreditCardModalContainer.js

```diff
@@ -88,7 +93,13 @@ class CreditCardModalContainer extends Component {
         stripeToken
       };
       const {error: cashayError} = await cashay.mutate('addBilling', {variables});
-      if (cashayError) throw new SubmissionError(cashayError);
+      if (cashayError) {
+        dispatch(
+          segmentEventTrack('addStripeBilling Cashay Error', {error: cashayError._error})
```

Comment:
i'd prefer to throw this one & the success one from the server for 2 reasons:
- less logic on the client
- guaranteed to call